tests/pipelines/test_pipelines_common.py

```diff
@@ -181,69 +87,166 @@ def __repr__(self):
         return f"ANY({', '.join(_type.__name__ for _type in self._types)})"
 
 
+def is_test_to_skip(test_casse_name, config_class, model_architecture, tokenizer_name, processor_name):
+    """Some tests are just not working"""
```

Comment:
Will have to check and try to fix progressively. To re-iterate: on current `main`, only fast tokenizer are used in pipeline tests (despite the generated tests have names like `_tokenizer_` and `_tokenizer_fast_`).